Sources/Crashes/CrashCollection.swift

```diff
@@ -80,8 +87,29 @@ public final class CrashCollection {
             didFindCrashReports(pixelParameters, processedData) {
                 Task {
                     for payload in processedData {
-                        await self.crashSender.send(payload)
+                        let crcid = self.crashCollectionStorage.object(forKey: Const.crcidKey) as? String
+                        let result =  await self.crashSender.send(payload, crcid: crcid)
+
+                        switch result.result {
+                        case .success:
+                            Logger.general.debug("Crash Collection - Sending Crash Report: succeeded")
+                            if let receivedCRCID = result.response?.allHeaderFields[CrashReportSender.httpHeaderCRCID] as? String {
+                                if crcid != receivedCRCID {
+                                    Logger.general.debug("Received new value for CRCID: \(receivedCRCID), setting local crcid value")
+                                    self.crashCollectionStorage.set(receivedCRCID, forKey: Const.crcidKey)
+                                } else {
+                                    Logger.general.debug("Received matching value for CRCID: \(receivedCRCID), no update necessary")
+                                }
+                            } else {
+                                Logger.general.debug("No value for CRCID header: \(Const.crcidKey), clearing local crcid value if present")
+                                self.crashCollectionStorage.removeObject(forKey: Const.crcidKey)
+                            }
+                        case .failure(let failure):
+                            // TODO: Is it worth sending a pixel for this case, so that we can monitor for missing crash reports?
```

Comment:
I think we should do this - we've had crashes go missing quietly when Apple moved to the IPS format on macOS, which was only detected after noticing how quiet the crash reports had been lately.

Reply:
Makes sense, I called it out because I thought it might be good idea but didn’t want to hold up starting to get feedback on my changes so far while I dug further.  Where should I look for a getting started guide / instructions / steps to follow for defining new pixels?  Presumably somewhere in Asana, but… asana search :(

Reply:
Yep, we do - check [this task](https://app.asana.com/0/69071770703008/1201077660132463/f) out. It'll tell you how to define a new pixel, which will likely lead you out to the Privacy Triage pixel definition form that'll ask you a range of questions.

Reply:
Pixel support added, but I’m not sure how to test it beyond watching client side logs.  Are there dashboards somewhere that show all our pixels?  Do I need to add monitoring for these new pixels somehow?